Sources/NIO/Channel.swift

```diff
@@ -337,6 +337,9 @@ public enum ChannelError: Error {
     /// address.
     case illegalMulticastAddress(SocketAddress)
 
+    /// Multicast is not supported on Interface
+    case multicastNotSupported(NIONetworkInterface)
```

Comment:
can we also add a test that triggers this and assert that the passed in interface is what we expect ?

Reply:
@normanmaurer we'd need an interface that doesn't support multicast for that, right?

Reply:
yeah